robottelo/cli/factory.py

```diff
@@ -2362,7 +2363,10 @@ def setup_cdn_and_custom_repos_content(
         # Upload the organization manifest
         try:
             manifests.upload_manifest_locked(
-                org_id, manifests.clone(), interface=manifests.INTERFACE_CLI
+                satellite=satellite,
+                org_id=org_id,
+                manifest=manifests.clone(),
```

Comment:
@mshriver we should use with statement whenever we are doing `manifests.clone()` or any such operation, right?
I am not sure, but could this be the reason why we are having issue with manifest?

Reply:
I don't believe the context manager functions for manifest are doing anything, I'll double check the use here. What I have is equivalent to master branch, just passing `satellite` through.